src/sidebar/components/MarkdownEditor.js

```diff
@@ -122,27 +127,27 @@ function ToolbarButton({
     tooltip += ` (${modifierKey}-${shortcutKey.toUpperCase()})`;
   }
 
+  const buttonProps = {
+    buttonRef,
+    disabled,
+    icon: iconName,
+    onClick,
+    tabIndex,
+    title: tooltip,
+  };
+
+  if (label) {
+    return (
+      <LabeledButton
+        classes="u-font--normal TransparentButton"
+        {...buttonProps}
+      >
+        {label}
+      </LabeledButton>
+    );
+  }
   return (
-    <button
-      className={classnames(
-        'MarkdownEditor__toolbar-button',
-        label && 'is-text'
-      )}
-      disabled={disabled}
-      onClick={onClick}
-      aria-label={tooltip}
-      title={tooltip}
-      tabIndex={tabIndex}
-      ref={buttonRef}
-    >
-      {iconName && (
-        <SvgIcon
-          name={iconName}
-          className="MarkdownEditor__toolbar-button-icon"
-        />
-      )}
-      {label}
-    </button>
+    <IconButton classes="hyp-u-padding--3 u-font--xsmall" {...buttonProps} />
```

Comment:
The size of the visible icons on touch screens has gotten somewhat smaller on this branch.

Master:

<img width="368" alt="Markdown toolbar main" src="https://user-images.githubusercontent.com/2458/141801728-0eeb65fe-c64b-4125-9859-91b5ec0dd224.png">

This branch:

<img width="447" alt="Markdown toolbar branch" src="https://user-images.githubusercontent.com/2458/141801767-40f818f8-80c8-40a5-9d72-5ad096d79235.png">

Is this known / intended?

Reply:
You're right; I've added a font-size change to the narrower-screen icons, making them bigger.